wasabi/research-club/anonymous-credentials.md

```diff
@@ -1,695 +1,327 @@
-You can, for example, besides range proofs, you can show that if you have two blinded tokens, that attributes are the same, for example.
+Unknown speaker  0:14  
+ Yes.
+
+Jonas  0:17  
+Or should be familiar for every Wasabi Power User, I guess. So the idea of credentials is similar in the sense that you get some blinded token that was signed by some server and the server does not see the message that was being signed. Now, the additional idea to a credential is that it's not only a message that is being signed, but instead, it's multiple attributes that can be signed. And these attributes can be anything. The interesting thing is that you can selectively reveal these attributes, you can say, here I have a token. And in this token there is this attribute, and it has this value. So the not standard example is you have a token from your government, let's say, and let's say birthdate is one attributes and then assigned. Now what you can do with that you can not only show your attributes, you can also show that it's in a certain range, you can do a range proof. So what you will do is if you go to the cinema, you don't have to reveal your age, you don't have to reveal anything else that is included in your credential, you only have to show that you are older than 18 years. And that works with such a credential. And these attributes are pretty flexible. You can for example, besides range proofs, you can show that if you have to blinded tokens that attributes are the same, for example. Yeah. So mathematically, I think the best way to think about this is as Peterson multi commitments. So in normal Peterson commitments, right, you have some randomness, and you use that to commit to a value. And you basically have two different group generators. Now with these multi commitments use in some of these credential approaches, you have randomness, and now you commit to multiple attributes and only a single one. And usually you have as many generators as they have attributes. So I think historically, one of the first notions of these credentials was by Brands, I think Peter Brands, not quite sure. He wrote his PhD. Yeah?
 
 ## Pedersen multi-commitments.
 
-Speaker 0: 00:02:17
+Unknown speaker  3:19  
+Stefano, I think, not Peter, or something like that.
 
-Yeah, so mathematically, I think the best way to think about this is as Peterson multi-commitments.
-So in normal Peterson commitments, right, you have some randomness and you use that to commit to a value.
-And you basically have two different generators, group generators.
-Now with these multi-commitments used in some of these credential approaches, you have randomness and now you commit to multiple attributes, not only a single one.
-And Usually you have as many generators as you have attributes.
-So I think historically one of the first notions of these credentials was by Branz.
-I think Peter Branz, not quite sure.
-He wrote his PhD.
-Yeah.
+Jonas  3:Unknown speaker5  
```

Comment:
You deleted the previous timestamps and altered the original formatting.

Reply:
Was I suppose to edit in the same job area?

Reply:
I am not sure that I understand your question. Based on the review guidelines you should keep the original formatting of the timestamps.